funcframework/framework.go

```diff
@@ -39,6 +39,9 @@ const (
 	fnErrorMessageStderrTmpl = "Function error: %v"
 )
 
+var errorVar error
+var errorType = reflect.TypeOf(&errorVar).Elem()
```

Comment:
Try this for one line? https://stackoverflow.com/questions/30688514/go-reflect-how-to-check-whether-reflect-type-is-an-error-type